pgtap/allpairs/floydWarshall-rows-check.sql

```diff
@@ -0,0 +1,87 @@
+\i setup.sql
+
+SELECT plan(20);
+
+SET extra_float_digits = -3;
+
+-- Check whether the same set of rows are returned always
+
+PREPARE expectedOutputDirected AS
+SELECT * FROM pgr_floydWarshall(
```

Comment:
Unfortunately you can not compare FLOAT just like even with the `extra_float_digits = -3;` that specially on the graph with the costs that are changed with very small float values. You need to transform the costs to text, so:

```
SELECT start_vid, end_vid, agg_cost::TEXT FROM pgr_floydWarshall(
```
Then as you can see in my experiment:
https://github.com/cvvergara/pgrouting/runs/874733075?check_suite_focus=true#step:7:373
The todo tests pass with the change

Reply:
Well, this was something new for me. Thanks! The `allpairs/` tests pass with this change.